its/projects/ProjectWithSourceGenerator/ProjectWithSourceGenAndAnalyzer/ProjectWithSourceGenAndAnalyzer.csproj

```diff
@@ -0,0 +1,19 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
[edu] why do you need a source generator for this test?

Reply:
That is how the functionality is actually tested, see : https://github.com/SonarSource/sonar-scanner-msbuild/pull/1257#discussion_r897774052